ADR/0009-pipeline-service-via-operator.md

```diff
@@ -0,0 +1,70 @@
+# 0009 Pipeline Service via Operator
+
+Created: 2023-09-29
+Last Updated: 2023-09-29
+
+## Status
+
+Proposed
+
+## Context
+
+kcp is no longer being used as a control plane for RHTAP. This means that
+"Pipeline Service" cannot be deployed as an independent service. For our
+initial MVP, all Tekton APIs need to be deployed onto a standard OpenShift
+cluster (specifically OpenShift Dedicated).
+
+## Decision
+
+All Tekton APIs will be provided using the stock OpenShift Pipelines operator (OSP).
+In the spirit of developing in a "Service First" manner, RHTAP will deploy
+a candidate "nightly" release of the operator. The service will be defined in
+the [pipeline-service](https://github.com/openshift/pipeline-service)
+repository, which is then imported into
+[infra-deployments](https://github.com/redhat-appstudio/infra-deployments) as
+an ArgoCD application.
+
+Configurations that are specific to RHTAP must be made available through
+the OSP CRDs. The following changes are specific to RHTAP:
+
+- Disable Tekton Triggers. Pipelines as Code will be the favored mechanism for
+  event-based triggering of pipelines for now. This decision can be revisited
+  in the future based on need.
```

Comment:
we are not yet doing this 

this is as much a "generic how you maintain ADRs" question, but I'll ask anway, as I see this in TEPs, KEPs ... I know there is the "proposed" status, and presumably when everything is done, the status becomes "implemented", but shouldn't we provide some indication of what bits are done and what bits are pending?

Reply:
I believe this question applies to other bullets here, but I'll refrain from making updates to each line

Reply:
My feeling is that the ADR should capture the destination, and the "current" state should be documented in the service itself.

Reply:
I agree with Romain - ADR is structured as "this is what we intend to do, here is the why and what is impacted."